test/lib/ensure-env-var.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+const test = require('ava')
+const Boom = require('boom')
+
+const lib = require('../../lib/ensure-env-var.js')
+
+test('Should throw Boom error if environment variables are not set', (t) => {
+  t.plan(2)
+  try {
+    lib('TEST')
+  } catch (err) {
+    t.deepEqual(err, Boom.badImplementation('TEST environment variable is mandatory'))
+  }
```

Comment:
Using catch so that i can test for an actual `Boom` error. I couldnt find a way to do this with `t.throws()`. This was the only way i could get the test to pass:
```js
t.throws(() => lib(), 'JWT_AUDIENCE environment variable is mandatory')
```